test/cpp/jit/test_gpu.cpp

```diff
@@ -4348,9 +4348,9 @@ void testGPU_FusionCacheBefore() {
 
   tv2->axis(0)->parallelize(ParallelType::BIDx);
   tv2->axis(-1)->parallelize(ParallelType::TIDx);
+  // Thread and Block binding
```

Comment:
nit: shouldn't this comment be next to the line defining the constants? (it seems that the empty line below the comment should be moved above it)